sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.props

```diff
@@ -13,7 +13,6 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
   <PropertyGroup>
     <FunctionsExecutionModel>isolated</FunctionsExecutionModel>
     <_FunctionsSkipCleanOutput>true</_FunctionsSkipCleanOutput>
-    <_IsFunctionsSdkBuild>true</_IsFunctionsSdkBuild>
```

Comment:
I am guessing this hardcoded value here was a copy-paste mistake. I do not see this being used anywhere else (except my new code in this PR).

Reply:
cc: @brettsam @fabiocav 

I agree with Shyju. Does this sound right?

Reply:
@kshyju might be worth pining Fabio via Teams to get an answer

Reply:
It should be OK to remove this, as we add a reference to the main SDK from here: https://github.com/Azure/azure-functions-dotnet-worker/blob/e7f8f127843f39d87df16a6611bccb54aff9a5a7/sdk/Sdk/ExtensionsCsprojGenerator.cs#L74, which is the project that cares about that.

We just need to make sure we validate the flow.

Reply:
Validated the flow of using a few worker extensions and everything looks good.